src/scrabble.js

```diff
@@ -1,5 +1,183 @@
-function scrabble() {
 
+//Core Scrabble: nailed it
+let newWordArr = []
+let sum = 0
+
+function scrabble(str) {
+    if (str === null) {
```

Comment:
That's a nice idea to check if there's a word first. That way you can have faster computing and probably less errors